ledger/heavy/executor/initial_state_keeper_test.go

```diff
@@ -0,0 +1,135 @@
+/*
+ * Copyright 2019 Insolar Technologies GmbH
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package executor
+
+import (
+	"testing"
+
+	"github.com/gojuno/minimock"
+	"github.com/insolar/insolar/insolar"
+	"github.com/insolar/insolar/insolar/gen"
+	"github.com/insolar/insolar/insolar/jet"
+	"github.com/insolar/insolar/insolar/record"
+	"github.com/insolar/insolar/instrumentation/inslogger"
+	"github.com/insolar/insolar/ledger/drop"
+	"github.com/insolar/insolar/ledger/object"
+	"github.com/stretchr/testify/require"
+)
+
+var (
```

Comment:
why not const?

Reply:
we cant get pointer to const

Reply:
Maybe we should just move this and other helper functions inside test function since its used only there.

Reply:
i hope there will be more tests lately)